src/components/view/ResourceView.vue

```diff
@@ -129,6 +148,26 @@ export default {
       } else {
         return true
       }
+    },
+    fetchResourceQuota () {
```

Comment:
@utchoang is there a way to not make changes in core components? Here's how you can extend the, for example;
- Create required slot in AutogenView etc
- Create a new custom component for rendering the view for a router name
- In the new/separate component you can re-use AutogenView and pass any specific component in the slots for example:
```
<AutogenView ...pass any params here.... >

<span slot="actions">
... define extension here, and methods relatd to that...
</span>

</AutogenView>
```

If you follow this pattern you won't need to just extend thing inside the core components of resourceview, autogenview, listview etc; but use slot to extend them.

Reply:
I've added a new component (QuotaSummaryResource) for view summary details and undo changed ResourceView.